src/deployments/cdk/src/deployments/iam/guardduty-roles.ts

```diff
@@ -73,7 +73,8 @@ export async function createAdminSetupRole(stack: AccountStack) {
         'guardduty:CreateMembers',
         'guardduty:UpdateOrganizationConfiguration',
         'guardduty:DescribeOrganizationConfiguration',
-        'guardduty:updateMemberDetectors',
+        'guardduty:UpdateMemberDetectors',
```

Comment:
I don't see "UpdateMemberDetectors" as a valid permission for GuardDuty.
![image](https://user-images.githubusercontent.com/41204211/100114374-394e4d80-2e26-11eb-8b82-4196be3d4a1f.png)

Reply:
@naveenkoppula - should this be UpdateDetector?

Reply:
After testing, "UpdateMemberDetectors" is the required permission.